roles/openshift_facts/library/openshift_facts.py

```diff
@@ -434,10 +435,17 @@ def set_aggregate_facts(facts):
         all_hostnames.add(facts['common']['public_hostname'])
 
         if 'master' in facts:
+            # FIXME: not sure why but facts['dns']['domain'] fails
```

Comment:
Yeah, I figured out it was hardcoded. I can change the comment to clearly indicate that if you'd like.